configuration/gfortran-debug.setup

```diff
@@ -11,14 +11,14 @@ if [[ "$uname" == "Linux"  ]] ; then
 		export NC4_DIR=/usr/local/ #singularity container only
 		export HDF_DIR=/usr/local/ #singularity container only
 elif [[ "$uname" == "Darwin" ]] ; then
-		export NC4_DIR="/usr/local/Cellar/netcdf/4.7.4_1"  #mac OS, brew install
-		export HDF_DIR="/usr/local/Cellar/hdf5/1.12.0_1"   #mac OS, brew install
+		export NC4_DIR=$(nf-config|awk 'BEGIN{FS="->\ "};/\-\-prefix/ {print $2}')  #mac OS (brew install)
```

Comment:
@gmao-cda thank you for this automation. Eventually this build approach will be deprecated in favor of cmake though.

Reply:
@StegmannJCSDA Thanks for sharing this information, Patrick!